JSDemos/Demos/TreeView/TreeViewWithSearchBar/description.md

```diff
@@ -1 +1,14 @@
-This demo demonstrates the TreeView component&rsquo;s searching capabilities. The component can find nodes containing your search string or&nbsp;only starting with it&nbsp;depending on&nbsp;the search mode.
\ No newline at end of file
+To add a search bar to the TreeView, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchEnabled) property to **true**.
+
+### Configure search properties
+* Search mode         
+To appear in the search results, a node's text should contain the search string. Use the [searchMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchMode) property to specify whether the text should start with or be equal to the search string instead. In this demo, you can use the drop-down list to switch between these modes.
```

Comment:
>To appear in the search results, a node's text should contain the search string.

I suggest inverting the point: 'The search returns nodes whose text contains the search string'. This seems a bit more straightforward.

>specify whether the text should start with or be equal to the search string instead

And what about the *"contains"* search mode?

Reply:
> And what about the "contains" search mode?

This sentence accounts for the "contains" mode: 'The search returns nodes whose text contains the search string'.